### ISSUE TITLE
Name on Phone

### ISSUE THREAD

**Issue by robertoamoreno** [jpwalters/OpenVBX-Plugin-SMS-Conversation#29]
It's great to be able to show the number as a name not sure if this works with addressbook.

**Comment by jpwalters** [comment]
I'm not aware that OpenVBX has an address book built into it does it?  Is there an address plugin?

**Comment by gegere** [comment]
Yes, I have been using this for several years and has been working very well. Maybe there can be a check for this. https://github.com/gegere/AddressBook-VBX
